features/intl-locale-info.yml

```
@@ -0,0 +1,13 @@
+# Merge with intl-locale.yml once this file's compat_features have reached baseline high
+
+name: Intl.Locale additions (week data, writing direction, hour cycle)
+description: "Additional getters in the `Intl.Locale` API provide supplemental locale information, such as calendar week data (first day in a week, weekend start day, weekend end day, minimum day in the first week), writing direction, 12- or 24-hour cycles, and numbering systems used in the locale."
```

"Additions" could probably go for _any_ future feature and we've previously only used parentheticals for disambiguation. In the absence of a name in use by developers, I've decided to follow the file name and proposal title.

(Also I'm struggling with how to describe this without obliging the reader to know what "locale" means, and failing. I've decided that this is a term of art with the definition owned by Unicode, so I've tried to mention Unicode to give the reader an idea of how to learn more.)

```suggestion
name: Intl.Locale info
description: The getter methods of the `Intl.Locale` API provide supplemental information about a Unicode locale, such as the locale's calendar (for example, the first day in a week or the weekend start day), writing direction, 12- or 24-hour cycles, and numbering system.
```